modules/ossm-configuring-distr-tracing-tempo.adoc

```diff
@@ -164,6 +162,29 @@ Kiali 1.73 uses the Jaeger Query API, which causes a longer response time depend
 
 . Verify the `istiod` pod logs and the `otel-collector` pod logs.
 
+[id="configuring-otel-collector-mtls-encrypted-namespace_{context}"]
+== Configuring the `OpenTelemetryCollector` in a mTLS encrypted Service Mesh member namespace
+
+All traffic is TLS encrypted when you enable Service Mesh `dataPlane` mTLS encryption.
+
+When the `OpenTelemetryCollector` is not part of the mesh, meaning there is no sidecar, this encryption is not expected.
```

Comment:
```suggestion

```

You don't need to mention this. Even if OTEL is part of the mesh, we still need to apply the following DestinationRule so that data is not encrypted to Tempo.